viseron/webserver/ui.py

```diff
@@ -0,0 +1,91 @@
+"""UI handlers."""
+import os
+
+import tornado.web
+
+from viseron.config import ViseronConfig, load_config
+from viseron.const import CONFIG_PATH
+from viseron.helpers import slugify
+
+
+class AboutHandler(tornado.web.RequestHandler):
+    """Handler for about-page."""
+
+    def get(self):
+        """GET request."""
+        self.render("about.html", version="maj.min.pat")
+
+
+class CamerasHandler(tornado.web.RequestHandler):
+    """Handler for cameras page."""
+
+    def get(self):
+        """GET request."""
+        config = ViseronConfig(load_config())
```

Comment:
We should provide the config either via API or maybe through a class variable on the Viseron class in `viseron/__init__.py`.

It could get kinda funky if you change the config and then reload the UI without restarting the container

Reply:
I think via API is a good choice. Imagine Home Assistant being able to just query the API and automatically configure _everything_ based on the provided configuration. 👍